packages/ra-ui-materialui/src/list/filter/FilterListItem.tsx

```diff
@@ -148,23 +149,21 @@ const FilterListItem: FC<{ label: string; value: any }> = props => {
     const translate = useTranslate();
     const classes = useStyles(props);
 
-    const isSelected = isEqual(value, filterValues);
```

Comment:
isEqual is wrong because we can have several filters in filterValues. We just want to know if "value" is part of filterValues, therefore using lodash isMatch instead